go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/grafana/grafana-plugin-sdk-go
 
-go 1.21
+go 1.22
```

Comment:
we should keep the same version than grafana: https://github.com/grafana/grafana/blob/main/go.mod#L3